src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcDumpDumper.cs

```diff
@@ -47,7 +47,7 @@ public ProcDumpDumper(IProcessHelper processHelper, IFileHelper fileHelper, IEnv
         _environment = environment;
     }
 
-    protected Action<object?, string> OutputReceivedCallback => (process, data) =>
+    protected Action<object?, string?> OutputReceivedCallback => (process, data) =>
```

Comment:
If the string is null, should we write a placeholder to the trace instead ? I'm talking about the `EqtTrace.Info` call below.